mmcv/ops/deform_conv.py

```diff
@@ -21,19 +21,18 @@ class DeformConv2dFunction(Function):
 
     @staticmethod
     def symbolic(g, input, offset, weight, stride, padding, dilation, groups,
-                 deform_groups, bias, im2col_step):
+                 deform_groups):
```

Comment:
It would be better to keep `bias` and `im2col_step` for future extension. Pls. keep them as default arguments